lib/normalize_country/countries/en.yml

```diff
@@ -2394,6 +2407,7 @@ SR:
 SS:
   aliases:
   - S. Sudan
+  - South Sudan, Republic of
```

Comment:
Do you think we may as well add `Republic of South Sudan` and `Republic of S. Sudan` and similar entries for the others. In English `Republic of South Sudan` is how it would be formally written.

Reply:
Sure. Although "S. Sudan" is a bit of a weird one, adding an extra permutation: abbreviations of "north", "south", etc. An "S." variation is included here and for a few other cases, but not for South Africa, South Georgia, North Macedonia etc.